### PR TITLE
Implements reduced fee ID eligibility logic.

### PR DESCRIPTION
**Pull request by rajivrajiv**
Implemented reducedFeeIdEligible().

### CODE REVIEW

File: src/site/static/eligibility.liquid [hunk 1]
```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
...
   function reducedFeeIdEligible() {
-    // TODO
-    return  false;
+    // https://www.dmv.ca.gov/portal/driver-licenses-identification-cards/identification-id-cards/
+    // https://www.icarol.info/ResourceView2.aspx?org=2225&agencynum=73919505
+    //
+    // DL 937 order form:
+    // https://www.dmv.ca.gov/portal/file/order-request-reduced-fee-or-no-fee-identification-card-program-dl-932-pdf/
+    //   Burton-Moscone-Bagley Citizens’ Income Security Act for Aged, Blind, and Disabled Californians (WIC §12000); is called WIC Section 12000 and does not have a seperate progam name, 
+    //   so could be part of the broader WIC benefits progam.
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2042]: The "WIC" in this case is the "welfare institutions code" and unassociated with the WIC program.  I believe the program described by WIC §12000 is SSP (the state portion of total SSI payments).

File: src/site/static/eligibility.liquid [hunk 2]
```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
...
+    // Other benefit programs for Aged, Blind and Disabled are: IHSS, SSI.
+    const isHoused = selectedOneOf('housing-situation', [
+      'housed',
+      'unlisted-stable-place']);
+    
+    const age = getValueOrNull('age');
+    const MAX_REDUCEDFEEID_ELIGIBLE_AGE = 62;  // Years
+
+    const isProgramQualified = or(
+      document.getElementById("existing-wic-me").checked,
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2053]: I think this should be removed, as the WIC referenced in the icarol.org and DL 937 documentation is different from the  Women, Infants and Children program.

File: src/site/static/eligibility.liquid [hunk 3]
```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
...
+      document.getElementById("existing-calworks-me").checked,
+      document.getElementById("existing-ihss-me").checked,
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2055]: I think SSI is the only aged, blind, and disabled program specifically referenced in the DL 937 order form, unless I've missed the reference to IHSS.

File: src/site/static/eligibility.liquid [hunk 4]
```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
...
+      document.getElementById("existing-ssi-me").checked,
+      document.getElementById('existing-ga-me').checked,
+      document.getElementById("existing-calfresh-me").checked,
+      document.getElementById("existing-cfap-me").checked,
+      document.getElementById("existing-capi-me").checked,
+      wicEligible(),
+      calworksEligible(),
+      ihssEligible(),
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2063]: Likewise, as above.

File: src/site/static/eligibility.liquid [hunk 5]
```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
...
+    const isProgramQualified = or(
+      document.getElementById("existing-wic-me").checked,
+      document.getElementById("existing-calworks-me").checked,
+      document.getElementById("existing-ihss-me").checked,
+      document.getElementById("existing-ssi-me").checked,
+      document.getElementById('existing-ga-me').checked,
+      document.getElementById("existing-calfresh-me").checked,
+      document.getElementById("existing-cfap-me").checked,
+      document.getElementById("existing-capi-me").checked,
+      wicEligible(),
```
Comment (trevorshannon) [inline src/site/static/eligibility.liquid:2061]: Likewise, as above.

**Review by trevorshannon — CHANGES_REQUESTED** [review]
Thanks very much!  The main request here is to remove IHSS and WIC from the list of programs that makes a person eligible for a reduced-fee ID.  Let me know if you disagree with that interpretation.

**Comment by rajivrajiv** [comment]
I will remove IHSS and WIC and a couple of comments. 

Why was WIC included in the benefits eligibility spreadsheet based on [ID Cards - California DMV](https://www.dmv.ca.gov/portal/driver-licenses-identification-cards/identification-id-cards/) ?

Thanks

**Review by trevorshannon — APPROVED** [review]
Looks good!

At the time I wrote that benefits eligibility spreadsheet, I _also_ assumed the "WIC" section referenced was having something to do with Women, Infants, and Children.  Since then, I've learned out about the Welfare Institutions Code.

**Pull request merged**